bika/lims/api/analysis.py

```diff
@@ -115,21 +115,20 @@ def get_formatted_interval(results_range, default=_marker):
         if default is not _marker:
             return default
         api.fail("Type not supported")
-
     results_range = ResultsRangeDict(results_range)
-    min_str = api.is_floatable(results_range.min) and results_range.min or ""
-    max_str = api.is_floatable(results_range.max) and results_range.max or ""
-    if not min_str and not max_str:
+    min_str = results_range.min if api.is_floatable(results_range.min) else None
+    max_str = results_range.max if api.is_floatable(results_range.max) else None
+    if min_str is None and max_str is None:
```

Comment:
Better use `if not all([min_str, max_str])`.

I think it would be a good idea to put the `api.is_floatable` test directly into the `ResultsRangeDict` and return `None` for these values from there directly.

Reply:
Thanks, I take note.